# Recommended literature

Here we have gathered some literature on metabarcoding.  
- [Zinger](Zinger_et_al_2019.pdf) et al. 2019. DNA metabarcoding—Need for robust experimental designs to draw sound ecological conclusions. Molecular Ecology, 28, 1857-1862
- [Deiner](Deiner_et_al-2017-Molecular_Ecology.pdf) et al. 2017. Environmental DNA metabarcoding: Transforming how we survey animal and plant communities. Molecular Ecology, 26, 5872-5895.
- [Bohmann](Bohmann_et_al_2014) et al. 2014. Environmental DNA for wildlife biology and biodiversity monitoring. TREE, 29.
- [Alberdi](Alberdi_et_al_2017.pdf) et al. 2017. Scrutinizing key steps for reliable metabarcoding of environmental samples. Methods in Ecology and Evolution, 9, 134-147.
- Ficetola et al. 2016. How to limit false positives in environmental DNA and metabarcoding? Molecular Ecology, 16, 604-607.
- [Dickie](Dickie_et_al_2018.pdf) et al. 2018. Towards robust and repeatable sampling methods ineDNA-based studies. Molecular Ecology Resources, 18, 940-952. Schnell et al. 2015. Tag jumps illuminated – reducing sequence‐to‐sample misidentifications in metabarcoding studies. Molecular Ecology Resources, 15, 1289-1303.
- [Antich](Antich_et_al_2021.pdf) et al. 2021. To denoise or to cluster, that is not the question: optimizing pipelines for COI metabarcoding and metaphylogeography. BMC Bioinformatics, 22,177.
- [Lamb](Lamb_et_al_2018.pdf) et al. 2019. How quantitative is metabarcoding: A meta‐analytical approach. Molecular Ecology, 28, 420-430.
- [Jamy](Jamy_et_al_2019.pdf) et al. 2019. Long-read metabarcoding of the eukaryotic rDNA operon to phylogenetically and taxonomically resolve environmental diversity. Molecular Ecology Resources, 20, 429-443.

### Methods
 - [Foster](Foster_et_al_2017.pdf) et al. 2017. Metacoder: An R package for visualization and manipulation of community taxonomic diversity data. PLoS Computational biology, 13.
- [Rognes](Rognes_et_al_2016.pdf) et al. 2016. VSEARCH: a versatile open source tool for metagenomics. PeerJ
- [Mahe](Mahe et al 2015.pdf) et al. 2015. Swarm v2: highly-scalable and high-resolution amplicon clustering. PeerJ
- [Callahan](Callahan_et_al_2016.pdf) et al. 2016. DADA2: High-resolution sample inference from Illumina amplicon data. Nature Methods, 13, 581-588.
- [Barbera](Barbera_et_al_2019.pdf) et al. 2019. EPA-ng: Massively Parallel Evolutionary Placement of Genetic Sequences. Systematic Biology, 68, 365-369.
